chrome/locale/en-US/options.dtd

```diff
@@ -45,14 +45,14 @@
 
 <!ENTITY display.label "Display">
 
-<!ENTITY showDKIMHeader.label  "Show DKIM header">
-<!ENTITY showDKIMStatusbarpanel.label  "Show DKIM status bar">
-<!ENTITY showDKIMFromTooltip.label  "Show DKIM tooltip for From header">
-<!ENTITY showDKIM.value.0.label  "never">
-<!ENTITY showDKIM.value.10.label "when an e-mail with a valid DKIM signature is viewed">
-<!ENTITY showDKIM.value.30.label "when an e-mail with a DKIM signature is viewed">
-<!ENTITY showDKIM.value.40.label "when an e-mail is viewed">
-<!ENTITY showDKIM.value.50.label "when a message is viewed">
+<!ENTITY showDKIMHeader.label  "Show DKIM header:">
+<!ENTITY showDKIMStatusbarpanel.label  "Show DKIM status bar:">
+<!ENTITY showDKIMFromTooltip.label  "Show DKIM tooltip for From header:">
+<!ENTITY showDKIM.value.0.label  "Never">
+<!ENTITY showDKIM.value.10.label "When an e-mail with a valid DKIM signature is viewed">
+<!ENTITY showDKIM.value.30.label "When an e-mail with a DKIM signature is viewed">
+<!ENTITY showDKIM.value.40.label "When an e-mail is viewed">
+<!ENTITY showDKIM.value.50.label "When a message is viewed">
```

Comment:
Unsure if capitalization is correct here (see also https://www.grammarly.com/blog/capitalization-after-colons/):
- for *never* I think it is wrong
- If we see the others both ways seem to be ok (depending what style you follow)
- For *error*/*warning* below seems to be ok as it is a noun. Question if it is mandatory or again just a style question?

Reply:
For consistency, I removed all capitalizations in English localization, even if it would be possible to use a capitalized word.